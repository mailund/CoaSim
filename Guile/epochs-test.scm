
(display (bottleneck 0.1 1 2))(newline)

(catch 'illegal-epoch
       (lambda () (bottleneck 0.1 2 1))
       (lambda (key . args) (display key)(newline)))

