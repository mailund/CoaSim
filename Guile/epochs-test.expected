(bottleneck 0.1 1.0 2.0)
illegal-epoch
illegal-epoch
